st2actions/st2actions/bootstrap/runnersregistrar.py

```diff
@@ -436,6 +436,31 @@
             }
         },
         'runner_module': 'st2actions.runners.windows_script_runner'
+    },
+    {
+        'name': 'cloudslang',
+        'aliases': [],
+        'description': 'A runner to execute cloudslang flows.',
+        'enabled': True,
+        'runner_parameters': {
+            'flow_path': {
+                'description': 'Path to a flow to run',
+                'type': 'string',
+                'required': True
+            },
+            'inputs': {
+                'description': ('Inputs which will be available to the execution'
```

Comment:
You are right. The reason I chose string is because the web UI didn't allow me to fill fields that aren't strings. Or that I missed something there..